ha/util/conf_store.py

```diff
@@ -110,3 +121,96 @@ def set_cluster_cardinality(self, index):
         cluster_cardinality_key = CLUSTER_CARDINALITY_KEY
         cluster_cardinality_value = {CLUSTER_CARDINALITY_NUM_NODES: num_pods, CLUSTER_CARDINALITY_LIST_NODES : watch_pods }
         self._confstore.update(cluster_cardinality_key, json.dumps(cluster_cardinality_value))
+
+    @staticmethod
+    def get_cvg_list(index, node_id):
+        """
+        Return list of CVG's for any given node.
+        Args:
+            index(str): index of conf file
+            node_id (str): node id
+        Returns:
+            list: list of CVG's
+
+        >>> ConftStoreSearch.get_cvg_list("cortx", '21d6291109304485b3daff43a06cff77')
+        ['cvg-01', 'cvg-02']
+        """
+        cvg_list = []
+        try:
+            cvg_count = Conf.get(index, GconfKeys.CVG_COUNT.value.format(node_id=node_id))
+            if cvg_count is None:
+                raise Exception(f"CVGs are not available for this node {node_id}")
```

Comment:
I don't think exception is needed here.  We can log an error message and continue.
@indrajitzagade  @shaileshsvc  any comments ?

Reply:
IMO, returning empty list would be fine. Let the client raise it on need.

Reply:
Logging exception message with Log.warn & returning empty list in this case.